### PR TITLE
Adding a draft for the debugkit new release

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: drafts/queries_log_for_your_ajax_requests.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Queries log for your Ajax requests
+==================================
+
+Have you ever needed to look inside what's going on with your AJAX requests?
+
+With the rise of javascript-heavy applications it is becoming a need to have
+more powerful tools for inspecting and debugging asynchronous requests without
+reloading the browser or checking special browser plugins.
+
+The new capabilities in `DebugKit`_ 3.1 make inspecting
```
Comment (jadb) [inline drafts/queries_log_for_your_ajax_requests.rst]: Am not sur eabout 'the new capabilities'.

Maybe just

> With DebugKit latest release (3.1), inspecting XHR requests is now possible.

**Comment by jadb** [comment]
:+1:

File: drafts/queries_log_for_your_ajax_requests.rst (hunk 1 above)
Comment (lorenzo) [inline drafts/queries_log_for_your_ajax_requests.rst]: ok :)

File: drafts/queries_log_for_your_ajax_requests.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Queries log for your Ajax requests
```
Comment (dereuromark) [inline drafts/queries_log_for_your_ajax_requests.rst]: AJAX

**Comment by markstory** [comment]
Put my comments as a commit. Looks great!

**Comment by lorenzo** [comment]
@markstory thanks!

**Pull request merged**